kong_api_authz/src/kong/plugins/opa/access.lua

```diff
@@ -50,13 +50,22 @@ function _M.execute(conf)
         local encoded_token = authorization:gsub("Bearer ", "")
         token = jwt:load_jwt(encoded_token)
     end
-
     -- input document that will be send to opa
-    local input = {
+	local input
```

Comment:
There seems to be some _tabs vs spaces_ action going on in this file 😄 let's keep it consistent, I think that means spaces. 😅